airbyte-integrations/connectors/source-greenhouse/source_greenhouse/components.py

```diff
@@ -25,7 +25,8 @@ def __post_init__(self, parameters: Mapping[str, Any]):
         self._state = {}
 
     def stream_slices(self) -> Iterable[StreamSlice]:
-        yield {self.request_cursor_field: self._state.get(self.cursor_field, self.START_DATETIME)}
+        slice = StreamSlice(partition={}, cursor_slice={self.request_cursor_field: self._state.get(self.cursor_field, self.START_DATETIME)})
```

Comment:
nit: if we're only yielding one slice in the subsequent line, can we just do this in one line instead

Reply:
Done 👍